vim-engine/src/main/kotlin/com/maddyhome/idea/vim/state/mode/editorExtensions.kt

```diff
@@ -26,6 +26,9 @@ val VimEditor.inSelectMode: Boolean
 val VimEditor.inCommandLineMode: Boolean
   get() = this.mode is Mode.CMD_LINE
 
+val VimEditor.inCommandLineModeWithVisual: Boolean
```

Comment:
Can you add a bit of docs explaining what this value should return? I mean, it's clear from the implementation, but let's explicitly state the intention of this value.
For example, if this function checks that "Mode is command" & "return is visual" – everything is ok. But if it means that return is visual for current command line mode, this function requires "inCommandLineMode" check before calling it, and an exception in case the mode is not CMD.

Reply:
Done!